docs/user_guide/HOWTO/Configure_Ssh.rst

```diff
@@ -1,118 +1,106 @@
+   Examples using Hiera are provided for illustrative purposes.
 
 In Hiera:
 
 .. code-block:: yaml
 
    ssh::server::conf::port: 2222
-   ssh::server::conf::ciphers:
-   - 'chacha20-poly1305@openssh.com'
-   - 'aes256-ctr'
-   - 'aes256-gcm@openssh.com
-
-In Puppet:
-
-.. code-block:: puppet
-
-   include 'ssh::server'
-
-   # Alternative:
-   # if `ssh::enable_server: true`, this will also work
-   include 'ssh'
+   ssh::server::conf::passwordauthentication: false
 
 
 Managing Additional Settings with ``sshd_config``
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-To manage SSH server settings that are not managed by the SIMP ``ssh`` module,
-use the ``sshd_config`` resource from `augeasproviders_ssh`_.  This is what the
-SIMP ``ssh`` module uses internally to manage the ``/etc/ssh/sshd_config``
-file, and you can use it to set any options ``ssh::server::conf`` does not
-manage.
-
-For instance, to set the sshd ``LogLevel`` option to ``VERBOSE``:
+Starting with version **6.4.0** of the ``simp-ssh`` module, you can use the
+`sshd_config`_ resource from the ``augeasproviders_ssh`` module to manage
+settings that the module does specifically define. To manage global sshd_config
+settings that are not included in the ``ssh::server::conf`` class, specify them
+in Hiera with the ``ssh::server::conf::custom_entries`` parameter as follows:
 
-.. code-block:: puppet
-
-   # VERBOSE will log SSH key fingerprints used for logins
-   sshd_config { 'LogLevel' : value => 'VERBOSE' }
-
-
-Mixing ``ssh::server::conf`` and ``sshd_config``
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-Some SSH server configurations may require a combination of
-``ssh::server::conf`` (for options that SIMP manages) and ``sshd_config``
-resources (for additional options). The following example configures the
-``/etc/ssh/sshd_config`` keys ``GSSAPIAuthentication``, ``GSSAPIKeyExchange``,
-and ``GSSAPICleanupCredentials`` with a value of "**yes**":
+.. NOTE::
 
-In Hiera:
+   Due to their complexity, ``Match`` entries are not supported and will
+   need to be added using ``sshd_config_match`` resources as described in
+   ``augeasproviders_ssh`` module.
 
 .. code-block:: yaml
 
-   # GSSAPIKeyExchange + GSSAPICleanupCredentials are managed via sshd_config
-   ssh::server::conf::gssapiauthentication: true
-
-In Puppet:
-
-.. code-block:: puppet
+  ssh::server::conf::custom_entries:
+    AllowAgentForwarding: "yes"
+    AuthorizedPrincipalsCommand: "/usr/local/bin/my_command"
+ 
+.. NOTE::
 
-   include 'ssh::server'
-
-   sshd_config {
-    default:
-      ensure => 'present',
-      value  => 'yes',
-    ;
-    # GSSAPIAuthentication is managed via `ssh::server::conf::gssapiauthentication`
-    ['GSSAPIKeyExchange', 'GSSAPICleanupCredentials']:
-      # use defaults
-    ;
-   }
+   This parameter is **not validated**. Be careful to only specify settings
+   that are only allowed for your particular SSH daemon and avoid duplicate
```

Comment:
Remove the redundant `only` by replacing `that are only allowed` with `that are allowed`.